clients/python/llmengine/data_types.py

```diff
@@ -163,17 +163,17 @@ class CreateLLMEndpointRequest(BaseModel):
     cpus: CpuSpecificationType
     gpus: int
     memory: StorageSpecificationType
-    gpu_type: GpuType
+    gpu_type: Optional[GpuType]
```

Comment:
hmm, why is this optional, will we deploy LLMs on CPU?

Reply:
offline: we're gonna add some defaults for the model architectures, I'm gonna keep this as optional since we may want overriding of defaults at some point

Reply:
for posterity: the `Optional` comes from the argument to `Model.create()`